src/v/raft/consensus.cc

```diff
@@ -2246,17 +2249,19 @@ ss::future<> consensus::refresh_commit_index() {
 }
 
 void consensus::maybe_update_leader_commit_idx() {
-    (void)with_gate(_bg, [this] {
-        return _op_lock.get_units().then(
-          [this](ss::semaphore_units<> u) mutable {
-              // do not update committed index if not the leader, this check has
-              // to be done under the semaphore
-              if (!is_leader()) {
-                  return ss::now();
-              }
-              return do_maybe_update_leader_commit_idx(std::move(u));
-          });
-    }).handle_exception([this](const std::exception_ptr& e) {
+    ssx::background = ssx::spawn_with_gate_then(_bg, [this] {
+                          return _op_lock.get_units().then(
+                            [this](ss::semaphore_units<> u) mutable {
+                                // do not update committed index if not the
+                                // leader, this check has to be done under the
+                                // semaphore
+                                if (!is_leader()) {
+                                    return ss::now();
+                                }
+                                return do_maybe_update_leader_commit_idx(
+                                  std::move(u));
+                            });
+                      }).handle_exception([this](const std::exception_ptr& e) {
```

Comment:
This formatting doesn't look right

Reply:
I think the rule is that in an assignment (in this case `background =`) it will try to indent the block to be right of the `=`.  Whereas when it wasn't an assignment, the block is only indented relative to the start of the line.

I say "I think" because I don't know the rules, I just let clang-format do its thing -- if we disagree with it here the PR will fail the lint check.